### DISCUSSION TITLE
Need help with GitHub Action failure with goreleaser

### DISCUSSION THREAD

**Discussion by jftuga** [goreleaser/goreleaser#2394] [Q&A] [answered]
If I run this locally, I can successfully build all of my binaries:

    goreleaser build --snapshot

I am getting a failure with my GH action and am not sure how to fix it. Any help would be greatly appreciated. Thanks.

https://github.com/jftuga/nics/runs/3317209083?check_suite_focus=true#step:4:60

**Comment by caarlos0** [comment, marked as answer]
```
  ⨯ release failed after 0.07s error=failed to build for darwin_arm64: exit status 1: go: cannot find main module, but found .git/config in /home/runner/work/nics/nics
61
	to create a module there, run:
62
	go mod init
```

The error is not on goreleaser, but on go build (which goreleaser calls)

**Reply by jftuga** [reply]
I fixed this. My `.gitignore` file was excluding `go.mod` and `go.sum`. After committing these files, the GH Action ran successfully. Thanks for your help.
